src/BlockExtractor.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace WP_CLI\I18n;
+
+use Gettext\Extractors\Extractor;
+use Gettext\Extractors\ExtractorInterface;
+use Gettext\Translations;
+use WP_CLI;
+
+final class BlockExtractor extends Extractor implements ExtractorInterface {
+	use IterableCodeExtractor;
+
+	public static $options = [
+		'translatableProperties' => [
+			'title',
+			'description',
+			'keywords',
+			'styles',
```

Comment:
I think we ended up with:
```suggestion
			'styleVariations',
```

to make it differ from `style` and `editorStyle`.